_posts/2020-05-30-Metal3-Cluster-Deployment.md

```diff
@@ -0,0 +1,544 @@
+---
+title: "Metal³ development environment walkthrough part 2: Deploying a new baremetal cluster"
+draft: false
+categories: ["metal3", "kubernetes", "cluster API", "metal3-dev-env"]
+author: Himanshu Roy
+---
+
+## Introduction
+
+This blog post describes how to deploy a bare metal cluster, a virtual one for simplicity, using [Metal³](https://github.com/metal3-io/metal3-dev-env). We will briefly discuss the steps involved in setting up the cluster as well as some of the customizations available. If you want to know more about the architechture of Metal³, this [blogpost](https://metal3.io/blog/2020/02/27/talk-kubernetes-finland-metal3.html) can be helpful.
+
+This post builds upon the detailed metal3-dev-env walkthrough that describes in detail the steps involved in the environment set up and management cluster configuration. Here we will use that environment to deploy a new Kubernetes cluster using Metal³.
```

Comment:
Link "the detailed metal3-dev-env walkthrough" to the post, i.e. `{% post_url 2020-02-18-metal3-dev-env-install-deep-dive %}`